clap_builder/src/builder/possible_value.rs

```diff
@@ -37,6 +39,7 @@ pub struct PossibleValue {
     name: Str,
     help: Option<StyledStr>,
     aliases: Vec<Str>, // (name, visible)
+    visible_aliases: Vec<Str>,
```

Comment:
Heh, that comment wasn't accurate when it was copied here but it shows that aliases are generally stored in a single `Vec`, rather than in parallel ones.  Let's do the same here.